architecture/backend/decisions-gherkin.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Architecture Decisions: Gherkin"
+nav_order: 1000
+parent: Backend
```

Comment:
Don't you want to put that under the "architecture" page you have just created?

Reply:
It's actually "Architecture" -> "Backend" -> Decisions, for when we'll add architecture about the frontend